exercism/diamond/diamond.js

```diff
@@ -0,0 +1,15 @@
+var Diamond = function() {}
+
+Diamond.prototype.makeDiamond = function(letter){
+  var alphabet = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
```

Comment:
I know this exercise isn't complete, but keep in mind that it might be case sensitive line 4 will work , but if the user inserts a lowercase letter will it still work? Use .toLowerCase() if it doesn't.